### PR TITLE
[Refactor] Import global var form vllm instead of overwirte it

### PR DESCRIPTION
**Pull request by MengqingCao**
### What this PR does / why we need it?
Import global var form vllm instead of overwirte it, so that we could use the correct global variant value

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
CI passed with existing test.

- vLLM version: v0.13.0
- vLLM main: https://github.com/vllm-project/vllm/commit/5326c89803566a131c928f7fdd2100b75c981a42

### CODE REVIEW

File: vllm_ascend/sample/logits_processor/__init__.py [hunk 1]
```diff
@@ -18,16 +18,7 @@
...
 
-# Error message when the user tries to initialize vLLM with a pooling model
-# and custom logitsproces
-STR_POOLING_REJECTS_LOGITSPROCS = ("Pooling models do not support custom"
-                                   " logits processors.")
-
-BUILTIN_LOGITS_PROCESSORS: list[type[LogitsProcessor]] = [
-    MinTokensLogitsProcessor,
-    LogitBiasLogitsProcessor,
-    AscendMinPLogitsProcessor,
```
Comment (MengqingCao) [inline vllm_ascend/sample/logits_processor/__init__.py:29]: check if this is indeed

**Review by weijinqian0 — APPROVED** [review]

File: vllm_ascend/sample/logits_processor/builtin.py [hunk 2]
```diff
@@ -1,52 +0,0 @@
-import torch
```
Comment (MengqingCao) [inline vllm_ascend/sample/logits_processor/builtin.py:1]: There are 2 reasons for overwriting `MinPLogitsProcessor`:
1. support pd-transfer in ascend scheduler scenario, introduced by #2753. As ascend scheduler is drop now, we don't need to overwirte the func `__init__` anymore
2. avoid indexing operation, which is already done in upstream vllm in https://github.com/vllm-project/vllm/pull/29826

Thus we could drop this safely
